Aspen-Discovery/aspen-discovery
# Sitemap Indexing

Fix issue where paths to exclude are not being excluded when using sitemaps 
Updated release notes

To test:
Create a website indexing setting using a sitemap for the url
Add some paths to exclude
Run the index and double check that they are excluded
